src/Templating/Templating.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace OpenIDConnectServer\Templating;
+
+class Templating {
+	private Loader $loader;
+
+	public function __construct() {
+		$this->loader = new Loader();
+	}
+
+	public function render( string $template_name, array $data = array() ): string {
+		$data['templates'] = $this;
+
+		ob_start();
+		$this->loader
+			->set_template_data( $data )
+			->get_template_part( $template_name );
+
+		return ob_get_clean();
```

Comment:
The intention was indeed to render the template to a string, and return that string, instead of echoing. This is a pattern I've encountered often and I feel it's appropriate, as sending the response to the client should not IMO be the responsibility of the templating engine. You could, for example, want to render a template into a `.html` file on disk, instead of sending it to the client.